addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -11,14 +11,15 @@
 
 from odoo import api, fields, models
 from odoo.addons.resource.models.resource import HOURS_PER_DAY
+from odoo.addons.hr_holidays.models.hr_leave import get_employee_from_context
 from odoo.exceptions import AccessError, UserError, ValidationError
 from odoo.tools.translate import _
 from odoo.tools.float_utils import float_round
 from odoo.tools.date_utils import get_timedelta
 from odoo.osv import expression
 
-_logger = logging.getLogger(__name__)
 
+_logger = logging.getLogger(__name__)
```

Comment:
![:sus:](https://media0.giphy.com/media/ANbD1CCdA3iI8/giphy.gif?cid=ecf05e47qb5idg7rn0bhk57vcmftk1ebgl5w0j7czhprd9w5&rid=giphy.gif&ct=g)

Reply:
It seems that there were a few unused _logger in hr_holidays... I removed them since they are not used :-)